### PR TITLE
Clean up tests so that they pass

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

**Comment by mrocklin** [comment]
Tests pass.  Merging soon if there are no comments.  Happy to change things later.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+git:
+    depth: false  # Ensure latest tag is pulled
+
+branches:
+    only:
+        - master
+
+env:
+    global:
+        - secure: "o4isEDi/jgj741PYg2GJDAVtLcr+rgnNUFyAKpTjD8C0zfYH7XK9Sf8M2+ymk2BsmVF47KxSI4L2Zzl86rUeSaGSz5UGPbI9MLlv+pge53Z7uPZAv015xxDCjzn44UfXYzG2G0nu0RZ8spX7wpf8fw6jYaBmKogwxxL2t2ys4PMgHZV/0z9AebUj9Byi125VmoE/nvvLeDogXiLkDrod/vRtltA/IrKaZiirXNSakeuY21mGM7KvU8crKC08xae2HBEUBzrYuW2lpULnxrDm/sOFOfumErcRalnCQVYjntvCKFneGgfsXXqp/JoLoxKaM19ACnW0jIfXJ/3BBo/AHtFeuLj+TE2kWtLVd1aNtmyMmElmmxiALLdTvclQ3UcPnkj0HavDnoppo3BXp7eQp4KFpxfIRETOpR6ZP/0gN+pQSEkMH6Ac8DHbDbqiXUN0quEMN0pbbl/78+sROfIaokHDxp6iqiTCpFKJBRnBBiz1C+36j+WKsVIT7YFYMt/keG3YqUmWj+pfXCL6Ghj4sUTHAgpSp2hSn6Ys6f4iwCpi8LC9O4pxsMJE7xGhPZqyP88XcLGt+R8D8HYrDuViCpSViLrIG66vwCmV92+2DwLWJOPQPXoJVtyZQz4qnfyAuig2rTqofIMMM0KtAWL6/0G+jUoZxds15Ymg5YbIRmE="
```
Comment (jcrist) [inline .travis.yml]: This is unnecessary.

File: continuous_integration/travis/run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+source activate test-environment
+py.test dask-yarn --verbose
+flake8 dask-yarn
```
Comment (jcrist) [inline continuous_integration/travis/run-tests.sh:5]: This script won't fail if tests don't fail. Needs a `set -xe` at the top.

File: continuous_integration/travis/install.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+conda config --set always_yes yes --set changeps1 no
+
+conda create -n test-environment \
+    cryptography \
+    dask \
+    distributed \
+    flake8 \
+    grpcio \
+    maven \
```
Comment (jcrist) [inline continuous_integration/travis/install.sh]: Maven is already installed on the image, shouldn't need to conda install it.

**Review by jcrist — COMMENTED** [review]
Thanks for a taking care of this. Left a few comments, but otherwise this looks good.

**Pull request merged**